### ISSUE TITLE
Any plan to use OpenAI Realtime API

### ISSUE THREAD

**Issue by aabdelrahmanm** [bolna-ai/bolna#37]
Real time voice conversation is in Beta and it could reduce the delay and enhance the experience, is there any plan to use OpenAI Real time voice? 

https://platform.openai.com/docs/guides/realtime

**Comment by prateeksachan** [comment]
Yes @aabdelrahmanm ! We'll be integrating and releasing it soon!

**Comment by AlexM4H** [comment]
Can you leave the issue open until the feature is integrated?

**Comment by buri1** [comment]
hey are there any updates on when the realtime api will be available?

**Comment by aabdelrahmanm** [comment]
That could be helpful https://github.com/openai/openai-realtime-twilio-demo

**Comment by 2softsoftware** [comment]
Best
